data/manual/default_gross_to_net_ratios.csv

```diff
@@ -0,0 +1,14 @@
+prime_mover_code,default_gtn_ratio
```

Comment:
Can we add the default/fallback value (.97) to this file so it's easier to find?

Reply:
I'll have to add entries for each prime mover, and maybe just add a column that indicates the data source.